blueocean-dashboard/src/main/less/run-details-header.less

```diff
@@ -12,23 +12,41 @@
 .RunDetailsHeader-times
 {
     height:100%;
-    padding-top: 1px;
+    padding-top: 2px;
 
     div {
-        margin-top: 8px;
-        min-height: 23px;
+        margin-top: 12px;
         display: flex;
         align-items: center;
 
         svg+* {
-            margin-left: 2px;
+            margin-left: 8px;
         }
     }
 }
 
 .RunDetailsHeader-sources,
 .RunDetailsHeader-times {
     flex-basis: 20%;
+    max-width: 20%;
+
+    .u-label-value {
+
+        label {
+            display: block;
+            width: 61px;
+            flex-shrink: 0;
```

Comment:
this is display block. i'm sure there are reasons. i *do* think we should stop using flex for everything, it is a bit buggy in certain situations from what i've seen, but for overall layouts great